src/modules/mc_hover_thrust_estimator/MulticopterHoverThrustEstimator.cpp

```diff
@@ -157,7 +157,22 @@ void MulticopterHoverThrustEstimator::Run()
 					ZeroOrderHoverThrustEkf::status status;
 					_hover_thrust_ekf.fuseAccZ(-local_pos.az, -local_pos_sp.thrust[2], status);
 
-					publishStatus(status);
+					hover_thrust_estimate_s status_msg;
```

Comment:
I would prefer if this is still in a function.

Reply:
Done.